recipe/meta.yaml

```diff
@@ -65,14 +64,14 @@ requirements:
     - setuptools
     - cudnn                                  # [cuda_compiler_version != "None"]
     - numpy
-    - pytorch
     - pytorch =*={{ torch_proc_type }}*
```

Comment:
Am wondering if this needs an update

@hmaarrfk what do you think?

Reply:
the cub?

Reply:
Meaning how GPU & CPU PyTorch packages are specified

Reply:
i think this is ok. we use this in torchvision